web-ui/src/components/team-member/TeamMemberContainer.jsx

```diff
@@ -0,0 +1,97 @@
+import React, { useContext, useState } from "react";
+import MemberIcon from "./MemberIcon";
+import ProfileContext from "../../context/ProfileContext";
+import "./TeamMember.css";
+
+const TeamMemberContainer = () => {
+  const context = useContext(ProfileContext);
+  // const profiles = context.teamMembers;
```

Comment:
can this commend be removed?

Reply:
That is the context that will be used when we don't have to mock the data anymore